src/components/borrewer/BorrowerDashboard.tsx

```
@@ -1,142 +1,80 @@
+        <div className="flex justify-between items-center mb-6  mt-4">
+          <div>
+            <h1 className="text-4xl font-bold bg-gradient-to-br from-green-400 to-blue-400 text-transparent bg-clip-text">{t("borrowerDashboard.title")}</h1>
+            <div className={`h-1 w-52 ${theme === "dark" ? "bg-[#38bdf8]" : "bg-[#0ea5e9]"} mt-2 rounded-full`}/>
+            <div className={`h-1 w-32 ${theme === "dark" ? "bg-[#38bdf8]" : "bg-[#0ea5e9]"} mt-2 rounded-full`}/>
+          </div>
+          <div>
+            <Button
+              variant="outline"
+              className={`py-4 px-6 rounded-3xl ${theme === "dark" ? "bg-[#0E1827] text-[#1DAFC9] border-[#0F2535] shadow-[0_4px_12px_rgba(1,242,160,0.15)]" : "bg-white text-[#0ea5e9] border-gray-200 shadow-md"}`}
+            >
+              <div className="h-4 w-4 rounded-full bg-gradient-to-r from-[#1DAFC9] via-[#38bdf8] to-[#10b981]"></div>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)